index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header>
+      <div class="nav-wrapper">
+        <nav class="navbar">
+          <div class="navtop">
+            <div class="time">
+              <h2>9:41</h2>
+            </div>
+            <div class="icons">
+              <i class="fa fa-signal" aria-hidden="true"></i>
+              <i class="fa fa-wifi" aria-hidden="true"></i>
+              <i class="fa fa-battery-full" aria-hidden="true"></i>
+            </div>
+          </div>
+          <div class="navbottom">
+            <div class="logo">
+              <h2>Carlyle Codes</h2>
+            </div>
+            <div class="menu">
+              <i class="fas fa-bars"></i>
+            </div>
+          </div>
+        </nav>
+      </div>
+      <div class="content-wrapper">
+        <div class="content">
+          <h1>Hey There. I'm Carlyle</h1>
+          <h3>I am a Software Developer</h3>
+          <h5>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h5>
+        </div>
+        <div class="social-links">
+          <img id="one" class="author" src="images/Vector@2x.png" alt="">
+          <i id="two" class="fab fa-linkedin-in"></i>
+          <img id="three" class="author" src="images/Vector.png" alt="two fingers, peace sign">
+          <img id="four" class="author" src="images/twitter.png" alt="twitter icon">
+          <img id="five" class="author" src="images/m-icon.png" alt="M">
+        </div>
```

Comment:
- [ ] Please consider using a `ul` tag to wrap all the social icons and use `li` tag to wrap each of the social icons instead of a `div` tag. This is because the social icons are unordered lists and therefore should be wrapped with a ul and li tag and also to improve the code structure.
- See my suggestion below:
```html
<ul class="social-links">
          <li><img id="one" class="author" src="images/Vector@2x.png" alt=""></li>
          <li><i id="two" class="fab fa-linkedin-in"></i></li>
          <li><img id="three" class="author" src="images/Vector.png" alt="two fingers, peace sign"></li>
          <li><img id="four" class="author" src="images/twitter.png" alt="twitter icon"></li>
          <li><img id="five" class="author" src="images/m-icon.png" alt="M"></li>
</ul>
```

- [ ] Please consider also adding the `a` tag to the social media icons. `<a>` is used to link from one page to another.

Reply:
I have tried adding the <a> but linter keeps throwing me this error; I have taken a screenshot for you:
<img width="575" alt="Screen Shot 2022-01-06 at 17 04 03" src="https://user-images.githubusercontent.com/61721428/148412694-5908d7f0-4bc9-4a2c-afea-f36edae33d2b.png">.

Reply:
We can't use `<a>` tag directly under `<ul>`,thats why you are getting this error use `<a>` thag like this example 👇 
`      <ul>
         <li>
          <a href="https://twitter.com">
             <i class="fa fa-twitter"></i>
          </a>
        </li>
     </ul>`